solvers/DPStokes/mobility.h

```diff
@@ -51,23 +52,58 @@ class DPStokes: public libmobility::Mobility{
     this->lanczosTolerance = ipar.tolerance;
     this->dppar.mode = this->wallmode;
     this->dppar.hydrodynamicRadius = ipar.hydrodynamicRadius[0];
-    this->dppar.w = 6;
-    this->dppar.beta = 1.714*this->dppar.w;
-    real h = this->dppar.hydrodynamicRadius/1.554;
+    this->dppar.w = 4;
+    this->dppar.beta = 1.785*this->dppar.w;
+    real h = this->dppar.hydrodynamicRadius/1.205;
     this->dppar.alpha = this->dppar.w/2.0;
     this->dppar.tolerance = 1e-6;
-    this->dppar.nx = int(this->dppar.Lx/h + 0.5);
-    this->dppar.ny = int(this->dppar.Ly/h + 0.5);
-    // Add a buffer of w*h/2 when there is an open boundary
+
+    real N_real = this->dppar.Lx/h; // actual N given L and h
+    int N_up = ceil(N_real);
+    int N_down = floor(N_real);
+    int N;
+    // either N_up or N_down will be a multiple of 2. pick the even one for a more FFT friendly grid.
+    if(N_up % 2 == 0){
+      N = N_up;
+    }else{
+      N = N_down;
+    }
+
+    this->dppar.nx = N;
+    this->dppar.ny = N;
+
+    // note: only set up for square boxes
```

Comment:
If this is a real restriction make it throw an exception when the use sends Lx!=Ly
```c++
if(this->dppar.Lx != this->dppar.Ly)
    throw std::runtime_error("[DPStokes] Only square boxes (Lx==Ly) are currently supported");
```